recipes/aws-c-cal/all/test_package/conanfile.py

```diff
@@ -13,5 +13,9 @@ def build(self):
 
     def test(self):
         if not tools.cross_building(self.settings):
+            stream = io.StringIO()
             bin_path = os.path.join("bin", "test_package")
-            self.run(bin_path, run_environment=True)
+            self.run(bin_path, run_environment=True, output=stream)
+            if bool(self.deps_user_info["aws-c-cal"].with_openssl):
+                assert "found static libcrypto" in stream.getvalue()
```

Comment:
```suggestion
                if not self.options["openssl"].shared:
                    assert "found static libcrypto" in stream.getvalue()
                else:
                    ...
```
You also need to fill in the `else` clausule.